xml/http_servers.xml

```diff
@@ -1766,9 +1766,10 @@
       <param pos="1" name="service.version"/>
    </fingerprint>
 
-   <fingerprint pattern="^PHP/(\S+)$">
-      <example>PHP/4.4.2-1build1</example>
-      <example>PHP/5.1.2</example>
+   <fingerprint pattern="^PHP/(\S+)(?:\s.+)?$">
+      <example service.component.version="4.4.2-1build1">PHP/4.4.2-1build1</example>
+      <example service.component.version="5.1.2">PHP/5.1.2</example>
+      <example service.component.version="5.3.14">PHP/5.3.14 ZendServer/5.0</example>
```

Comment:
IMO, just match ZendServer.  Catch-alls are nice in that they will match in unknown situations, but are bad in that the handling of these unknown situations may need improvement and the catch all could hide it.

Take OpenSSH recently, as an example.  There were Ubuntu-version specific fingerprints for OpenSSH, and then a generic OpenSSH catch-all.  Unfortunately, there were other Ubuntu OpenSSH versions that we did not have an explicit match for which were being matched by the catch all, resulting in us missing the opportunity for getting an OS fingerprint.